src/components/homeWidgets/TodaysBirthdaysWidget.tsx

```diff
@@ -0,0 +1,28 @@
+import styles from './TodaysBirthdaysWidget.module.scss';
+import { useAppTranslation } from '@/lib/i18n';
+import { WidgetLayout } from '@/components/homeWidgets/WidgetLayout';
+import useTodaysBirthdays from '@/api/users/getTodaysBirthdays';
+
+export default function UEBrowserWidget() {
+  const { t } = useAppTranslation();
+  const users = useTodaysBirthdays();
+  return (
+    <WidgetLayout
+      title={t('homepage:todaysBirthdays.title')}
+      subtitle={t('homepage:todaysBirthdays.subtitle')}
+      className={styles.widget}>
+      {users !== null
+        ? users.length === 0
+          ? "Aucun anniversaire aujourd'hui"
+          : users.map((user) => (
+              <div key={user.id} className={styles.user}>
+                <div className={styles.name}>
+                  {user.firstName} {user.lastName}
+                </div>
+                <div className={styles.age}>{t('homepage:todaysBirthday:age', { age: user.age })}</div>
```

Comment:
C'est pas plutôt un point ? *Et selon si tu rajoutes un `s` à `todaysBirthday` faudra ptet aussi modifier ici*
```suggestion
                <div className={styles.age}>{t('homepage:todaysBirthday.age', { age: user.age })}</div>
```